bigframes/clients.py

```diff
@@ -172,10 +173,9 @@ def _ensure_iam_binding(
                     return
 
         # Create a new binding
-        new_binding = {
-            "role": role,
-            "members": [service_account],
-        }  # Use a dictionary to avoid problematic google.iam namespace package.
+        new_binding = google.iam.v1.policy_pb2.Binding(
+            role=role, members=[service_account]
+        )  # Use a dictionary to avoid problematic google.iam namespace package.
```

Comment:
This comment is now out-of-date.

```suggestion
        )
```

Could you try this in google3? I wonder if a `//google/iam/v1:iam_policy_py_pb2` dependency was missing last time I tried it?

Reply:
Trying. That one should be indirectly included via resourcemanager_v3 (which does include it)?